### ISSUE TITLE
Invisible cursor.

### ISSUE THREAD

**Issue by CzravissSzkot** [WillPower3309/swayfx#351]
- **Description:**
  - The steps you took in plain English to reproduce the problem.
sway version : 0.4.1 (aur)
system : Arch 6.10.10-arch1-1 x86_64
hardware : intel I7 7700k / Nvidia GTX 1080
drivers : nouveau Nvidia open source drivers

Steps :
1. use archinstall to install arch + sway with ly greeter
2. Install eww, yay, etc.
3. try to add animations
4. see SwayFX
5. Install it via AUR package
6. after install reboot and login to SwayFX session
7. Notice there is no cursor (from the get go)
8. Go back to sway and write this issue.

any ides what could be the issue of the cursor not existing

note : you can use mouse you just dont see the cursor.

**Comment by danielwis** [comment]
You might need to set `WLR_NO_HARDWARE_CURSORS=1` (run swayfx using `WLR_NO_HARDWARE_CURSORS=1 sway`)
